packages/p2p/src/utils/date-time.js

```diff
@@ -7,6 +7,14 @@ export const getFormattedDateString = (date_obj) => {
     return `${day} ${month} ${year} ${time}`;
 };
 
+export const getLocalUnix = (epoch) => {
+    if (typeof epoch !== 'number') throw Error('getLocalEpoch argument needs a number');
+
+    const local_unix = epoch * 1000;
```

Comment:
why are we calling it local_unix? it sounds like it means local time, but we're just turning it to miliseconds. let's call it something like convertToMS or something

Reply:
@mustofa-binary This is still pending.